CONTRIBUTING.md

```diff
@@ -60,7 +60,7 @@ In addition, blind users in particular navigate by link text, so a blind user wi
 
 ### D3. Remove unnecessary clicks
 
-Whenever you give a user a button to click, always think "in what situations would a user want to click this? in what situations would a user not want to click this?" Dialogs like "Are you sure?" can often be replaced with just doing the thing with an "Undo" button. Buttons to show more details can often be replaced with simply showing more details by default.
+Whenever you give a user a button to click, always think "In what situations would a user want to click this? In what situations would a user not want to click this?" Dialogs like "Are you sure?" can often be replaced with just doing the thing with an "Undo" button. Buttons to show more details can often be replaced with simply more details by default.
```

Comment:
So there are two ways to handle quotes, depending on whether or not they're complete sentences:

> Nails? Think "hammer".

> Nails? Think, "A hammer would be useful here."

I'm using the first way here, which is complicated because we _are_ using multiple sentences. But if you're going to switch to complete sentence style, you should fully switch to complete sentence style.

As for "showing", it's definitely clearer with it than without it. I don't see any grammatical error with that one.

```suggestion
Whenever you give a user a button to click, always think, "In what situations would a user want to click this? In what situations would a user not want to click this?" Dialogs like "Are you sure?" can often be replaced with just doing the thing with an "Undo" button. Buttons to show more details can often be replaced with simply showing more details by default.
```

Reply:
I interpreted a disagreement between gerunds, but I did not interpret "Buttons to show more details" as a gerund phrase rather than just "Buttons" itself. Thank you for pointing out the clarity issue, though, as that is most important.